lib/preview/link.js

```diff
@@ -36,6 +36,11 @@ var PreviewConstants = require('./constants');
 var PreviewImage = require('./image');
 var Result = require('./model').Result;
 
+// URLs matching any of these regexes will be marked "not embeddable" over all protocols.
+var DISALLOW_EMBED_REGEXES = [
+  /^http(s)?:\/\/(www\.)?portfolium\.com\/entry/
```

Comment:
@johncrossman I believe the `portfolium.cloudimg.io` URLs are for stored images, which don't present the problem addressed here (of Portfolium entry URLs confusingly reproducing the entire Portfolium navigation inside SuiteC). It's possible that we'll have to account for some other URL format at some point, but that will depend on Oliver or others defining a workflow.